README.md

```diff
@@ -36,9 +36,20 @@ SilverStripe\GarbageCollector\GarbageCollectorService:
   collectors:
     - 'SilverStripe\GarbageCollector\Collectors\VersionedCollector'
     - 'SilverStripe\GarbageCollector\Collectors\ChangeSetCollector'
+
+#Queue a RecurringAllGarbageCollectorJob if there isn't one already
+Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor:
+  extensions:
+    - 'SilverStripe\GarbageCollector\Extensions\QueuedJobDescriptorExtension'
+
+#Automatically run all garbadge collectors every day
+SilverStripe\GarbageCollector\Jobs\RecurringAllGarbageCollectorJob:
+  seconds_between_jobs: 86400
```

Comment:
Can you please put this directly into the class? Then it can be changed via yaml, but the default should be in the class itself.